types/twitter-autohook/index.d.ts

```diff
@@ -0,0 +1,43 @@
+// Type definitions for twitter-autohook 1.7
+// Project: https://github.com/twitterdev/autohook#readme
+// Definitions by: David Jiménez <https://github.com/dubisdev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import EventEmitter = require('events');
+
+export class Autohook extends EventEmitter {
+    constructor(configuration: {
+        token: string;
+        token_secret: string;
+        consumer_key: string;
+        consumer_secret: string;
+        ngrok_secret?: string;
+        env: string;
+        port: number;
+        headers?: any[];
+    });
+
+    startServer(): void;
+    setWebhook(webhookUrl: string): Promise<unknown>;
```

Comment:
```suggestion
    /** @async */
    setWebhook(webhookUrl: string): Promise<unknown>;
```
https://jsdoc.app/tags-async.html